docs/src/prototypes/chatMessages/ThreadedMessages/ThreadReplies.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  AvatarProps,
+  Avatar,
+  ChatMessageProps,
+  Button,
+  ChatItem,
+  ChatItemProps,
+  Chat,
+} from '@stardust-ui/react'
+import repliesButtonBehavior from './repliesButtonBehavior'
+import ScreenReaderHeaderText from './SreenReaderHeaderText'
+
+export type ThreadReplyProps = ChatMessageProps & {
+  avatar?: AvatarProps
+}
+
+type ThreadRepliesProps = {
+  replies?: ThreadReplyProps[]
+}
+
+const ThreadReplies = (props: ThreadRepliesProps) => {
+  const [expanded, setExpanded] = React.useState(false)
+
+  const renderCollapseAllOrRepliesButton = () => {
```

Comment:
This thing looks like a separate component, something like `ThreadCollapseButton`

Reply:
This button is always rendered, only the text is different. I'll rename the method